annotation/annotation/models.py

```diff
@@ -200,6 +200,17 @@ class User(Base):
         "Job", secondary=association_job_owner, back_populates="owners"
     )
 
+    def __eq__(self: "User", other: object) -> bool:
```

Comment:
never type self, excessive

Reply:
you have to have self in any method, but you shouldn't specify typing for self, it is obvious the same as the class in which this method is located